IaC/PnP/monitoring/aks/online/aks_secure_baseline/ngsa-memory-with-crd/README.md

```diff
@@ -0,0 +1,82 @@
+# Deploying NGSA Memory with Traefik middleware CRF
```

Comment:
Is this CRF or CRD? If it's CRD, let's use the full name here (CustomResourceDefinition?)

Reply:
Fixed typo. But using CRD. Using full form makes the section title long. It's common Kubernetes term.